app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  # Associations
+  # A like belongs to an author
+  # A like belongs to a post
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+end

```

Comment:
- [ ] Kindly add in this Like model, a method that updates the likes counter for a post. 👍🏽

**It would be nice to have the after_save here as well** 👍🏽